docs/source/asr/speaker_diarization/configs.rst

```diff
@@ -1,73 +1,118 @@
 NeMo Speaker Diarization Configuration Files
 ============================================
 
-Since speaker diarization model here is not a fully-trainble End-to-End model but an inference pipeline, we use **diarizer** instead of **model** which is used in other tasks.
+Since speaker diarization model here is not a fully-trainable End-to-End model but an inference pipeline, we use **diarizer** instead of **model** which is used in other tasks.
 
-The diarizer section will generally require information about the dataset(s) being
-used, models used in this pipline, as well as inference related parameters such as postprocessing of each models.
+The diarizer section will generally require information about the dataset(s) being used, models used in this pipeline, as well as inference related parameters such as post processing of each models.
 The sections on this page cover each of these in more detail.
 
-
-Example configuration files for speaker diarization can be found in the
-``<NeMo_git_root>/examples/speaker_tasks/diarization/conf/speaker_diarization.yaml>``
+Example configuration files for speaker diarization can be found in ``<NeMo_git_root>/examples/speaker_tasks/diarization/conf/offline_diarization.yaml``
 
 .. note::
-  For model details and deep understanding about configs, finetuning, tuning threshold, and evaluation, 
+  For model details and deep understanding about configs, fine-tuning, tuning threshold, and evaluation, 
   please refer to ``<NeMo_git_root>/tutorials/speaker_tasks/Speaker_Diarization_Inference.ipynb``;
   for other applications such as possible integration with ASR, have a look at ``<NeMo_git_root>/tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb``.
 
 
 Dataset Configuration
------------------------
+---------------------
 
 In contrast to other ASR related tasks or models in NeMo, speaker diarization supported in NeMo is a modular inference pipeline.
-Datasets here denotes the data you would like to perform speaker diarization on. 
+Datasets here denote the data you would like to perform speaker diarization on. 
```

Comment:
The "datasets" here is outdated. Please update

Reply:
Fixed this sentence since this is somewhat misleading